curves/LICENSE-APACHE

```diff
@@ -0,0 +1,201 @@
+                              Apache License
```

Comment:
Let's make this a pointer to the top level `LICENSE-APACHE`

Reply:
are symlinks fine? I forget if there's a GitHub specific way to do this